plugins/offboard/offboard_impl.cpp

```diff
@@ -160,6 +234,101 @@ void OffboardImpl::set_velocity_body(Offboard::VelocityBodyYawspeed velocity_bod
     send_velocity_body();
 }
 
+void OffboardImpl::send_translation_ned()
+{
+    // by default ingore all setpoints
```

Comment:
I think this comment is wrong because that's not what we're doing.
Also, `ingore` -> `ignore`.